ui/.env.development

```diff
@@ -1 +1,2 @@
 REACT_APP_MIXPANEL_TOKEN=6b8d3fb93ffd04f2e76325540c838624
+REACT_APP_MIXPANEL_HOST=http://127.0.0.1:8090/api/v1/dashboard/report
```

Comment:
Is it better to not provide a local endpoint in development? Otherwise developers need to set up a local server in order to see what kind of event is collected, instead of just using the Mixpanel UI.

Reply:
Make sense, but this doesn't affect the 4.0.2 release. I will update it after finishing the telemetry-log-collector, and when developing, I can put it in the local .env file.

Reply:
Yup. I think so.